README.md

```diff
@@ -11,6 +11,18 @@ in pure vim script with python dependency removed.
 Plug 'prabirshrestha/asyncomplete.vim'
 ```
 
+To enable default mappings use the following setting. This is disabled by default.
```

Comment:
I actually preferred opt-out instead of opt-in by default. But given that people customize vim a lot I didn't think it would be a good idea. I'm still open of opt-out.